cvat-canvas/src/typescript/drawHandler.ts

```diff
@@ -96,9 +96,9 @@ export class DrawHandlerImpl implements DrawHandler {
             Vertical,
         }
 
-        const isBetween = (x1: number, x2: number, c: number): boolean => (
-            c >= Math.min(x1, x2) && c <= Math.max(x1, x2)
-        );
+        function isBetween(x1: number, x2: number, c: number): boolean {
+            return c >= Math.min(x1, x2) && c <= Math.max(x1, x2);
+        }
```

Comment:
Conflict between prettier and eslint. The first removes parentness, the second removes extra line and gets too many characters in the line (> 120) => error. No ways to configure prettier ((

Reply:
One way to solve is to transform arrow function to general function